f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -723,21 +703,83 @@ Igce.prototype = {
             if (igceInstructions.surge_requirement_capabilities === "YES") {
                 igceSurgeCapabilities = igceInstructions.surge_requirement_capacity;
             }
-            var toolsUsed = "";
-            if (igceInstructions.how_est_dev_tools_used) {
-                toolsUsed += igceInstructions.how_est_dev_tools_used;
+
+            //expected to be CSV w/ values AWS/GOOGLE_CLOUD/MICROSOFT_AZURE/ORACLE_CLOUD/PREVIOUSLY_PAID_PRICES/OTHER
+            //source: EstimatesDeveloped.vue
+            var toolsUsed = igceInstructions.how_est_dev_tools_used;
+            toolsUsed = toolsUsed
+                .replace("AWS", "AWS pricing calculator")
+                .replace("GOOGLE_CLOUD", "Google Cloud pricing calculator")
+                .replace("MICROSOFT_AZURE", "Microsoft Azure pricing calculator")
+                .replace("ORACLE_CLOUD", "Oracle Cloud pricing calculator")
+                .replace("PREVIOUSLY_PAID_PRICES", "Prices previously paid on a same or similar contract")
+                .replace("OTHER", igceInstructions.how_est_dev_other_tools_used);
+            //discard leading commas
+            if (toolsUsed.startsWith(',')) {
+                while (toolsUsed.charAt(0) === ',') {
+                    toolsUsed = toolsUsed.substring(1);
+                }
             }
-            if (igceInstructions.how_est_dev_other_tools_used) {
-                toolsUsed += " " + igceInstructions.how_est_dev_other_tools_used;
+
+            var previousEstimateComparison = "";
+            if (igceInstructions.how_est_dev_prev_cost_estimate_comp_option) {
+                switch (igceInstructions.how_est_dev_prev_cost_estimate_comp_option) {
+                    case "MORE_THAN":
+                        previousEstimateComparison = "The previous cost estimate was more than the actual prices paid.";
+                        break;
+                    case "LESS_THAN":
+                        previousEstimateComparison = "The previous cost estimate was less than the actual prices paid.";
+                        break;
+                    case "SAME":
+                        previousEstimateComparison = "The previous cost estimate was approximately the same as the actual prices paid.";
+                        break;
+                }
             }
 
             instructions = {
                 estimateDescription: igceInstructions.how_est_dev_cost_estimate_description,
-                assumptionsMade: "assumptions",
+                assumptionsMade: "not used",
                 toolsUsed: toolsUsed,
-                informationSource: "info source",
-                previousEstimateComparison: igceInstructions.how_est_dev_prev_cost_estimate_comp_option
+                informationSource: "not used",
```

Comment:
For clarity; corresponding rows have been removed from IGCE Instructions sheet in output

Reply:
That is correct.